integration-tests/docker-compose.yaml

```diff
@@ -0,0 +1,62 @@
+version: "3"
+services:
+
+  mimir:
+    image: grafana/mimir:2.9.0
+    volumes:
+      - ./configs/mimir:/etc/mimir-config
+    entrypoint:
+      - /bin/mimir
+      - -config.file=/etc/mimir-config/mimir.yaml
+    ports:
+      - "9009:9009"
+
+  loki:
+    image: grafana/loki:2.8.3
+    command: -config.file=/etc/loki/local-config.yaml
+    ports:
+      - "3100:3100"
+
+  otel-collector:
```

Comment:
I'm not sure if we should be running a Collector. I think we should just test the Agent. But I'm ok to leave this here for now.

Reply:
Yeah I wanted to use it for exemplars but I went for a more straightforward test. I would be happy to reconsider it in another issue